builder/builder.go

```diff
@@ -44,8 +44,8 @@ import (
 	"github.com/pkg/errors"
 )
 
-// Version of Mixer. Also used by the Makefile for releases.
-const Version = "5.1.0"
+// Version of Mixer. This is provided by ldflags in Makefile during compilation
+var Version = ""
```

Comment:
This is used in https://github.com/clearlinux/mixer-tools/blob/master/mixer/cmd/root.go#L56

so mixer --version will print "" if this is left empty.

Reply:
This won't be empty. The value is set on compile time by `-ldflags="-X ${GO_PACKAGE_PREFIX}/builder.Version=${VERSION}"`
Doccumentation regarding -X option is pretty bad, but this seems right.

Reply:
[Line #40](https://github.com/clearlinux/mixer-tools/pull/401/files/288c5be3ce7215f1b0e5ffa8ff524fefe419f862#diff-b67911656ef5d18c4ae36cb6741b7965R40) in Makefile provides the value for Version using ldflags. I agree documentation is bad.